chat/src/main/java/io/skygear/plugins/chat/ui/model/User.kt

```diff
@@ -34,16 +40,26 @@ class User : IUser {
         this.chatUser = u
     }
 
-    override fun getId(): String {
-        return this.chatUser.id
-    }
+    override fun getId() = this.chatUser.id
 
     override fun getName(): String {
-        return this.chatUser.record.get(User.DisplayNameField) as String
+        val userName = this.chatUser.record.get(User.DisplayNameField) as String?
+        if (userName != null) {
+            return userName
+        }
+
+        return User.DefaultDisplayName
     }
 
     override fun getAvatar(): String {
-        return this.chatUser.record.get(User.AvatarField) as String
+        val avatarUrl = this.chatUser.record.get(User.AvatarField) as String?
```

Comment:
As I remember the AvatarField should be assets instead of string? I think we should write down the assumption like user profile need to have `name` as string and the avatar fields. Besides that, the PR looks fine. Let me merge it first and I will open issue to see if we need to change it to assets :)